### PR TITLE
[grafana] Fix "Disk space usage" panel for btrfs

### PR DESCRIPTION
**Pull request by witekest**
Size of btrfs subvolumes is not reported by node exporter. Only usage data on the device level is meaningful for visualisation.

### CODE REVIEW

**Review by deneb-alpha — CHANGES_REQUESTED** [review]
please, do a run with spec-cleaner too. At least the copyright year needs an update. 

About the other changes,  I didn't spot anything wrong

**Comment by witekest** [comment]
> please, do a run with spec-cleaner too. At least the copyright year needs an update.

Thanks for catching. Done.

File: grafana-formula/grafana-formula.spec [hunk 1]
```diff
@@ -44,7 +43,6 @@ cp -R grafana/* %{buildroot}%{fdir}/states/%{fname}
 cp -R metadata/* %{buildroot}%{fdir}/metadata/%{fname}
 
 %files
-%defattr(-,root,root)
```
Comment (deneb-alpha) [inline grafana-formula/grafana-formula.spec:47]: same here. spec-cleaner is sometimes removing too much. not sure  if this is wanted or  no

File: grafana-formula/grafana-formula.spec [hunk 2]
```diff
@@ -19,14 +19,13 @@
...
-Version:        0.10.0
+Version:        0.10.1
 Release:        0
 Summary:        Salt formula for installing and configuring Grafana
 License:        Apache-2.0
 Group:          System/Packages
-Url:            https://github.com/SUSE/salt-formulas
+URL:            https://github.com/SUSE/salt-formulas
 Source:         grafana-formula-%{version}.tar.gz
-BuildRoot:      %{_tmppath}/%{name}-%{version}-build
```
[the remark this answers is not in the document]
Comment (witekest) [reply grafana-formula/grafana-formula.spec:29]: `BuildRoot` tag is obsolete and seems not to have any effect on the build.

File: grafana-formula/grafana-formula.spec (hunk 1 above)
Comment (witekest) [reply grafana-formula/grafana-formula.spec]: Indeed, I think it's safer to keep this one. I reverted the change.

**Review by deneb-alpha — APPROVED** [review]
LGTM!

**Pull request merged**